cmd/snapd/main.go

```diff
@@ -64,6 +90,12 @@ func run() error {
 
 	d.Start()
 
+	if ticker, err := runWatchdog(d); err != nil {
+		logger.Noticef("cannot run software watchdog: %s", err)
+	} else {
+		defer ticker.Stop()
```

Comment:
Thanks! You are right about the notify, this is silly because if snapd fails to check-in with systemd, it will be killed anyway. The ticker is stopped in a defer (just like the daemon is stopped). But I chagned the code now to make it more obvious.